sos/policies/redhat.py

```diff
@@ -200,7 +205,9 @@ def get_tmp_dir(self, opt_tmp_dir):
 
 CB = "cantboot"
 CB_DESC = "For use when normal system startup fails"
-CB_OPTS = SoSOptions(verify=True, all_logs=True, onlyplugins=_cb_plugs)
+CB_OPTS = SoSOptions(verify=True, all_logs=True, onlyplugins=_cb_plugs,
+                     plugopts=['boot.all-images=on', 'rpm.rpmva=on',
+                     'rpm.rpmdb=on'])
```

Comment:
This is where the Travis failure is coming from. The list continuation needs to be indented to the same level, e.g.:

```python
CB_OPTS = SoSOptions(verify=True, all_logs=True, onlyplugins=_cb_plugs,
                     plugopts=['boot.all-images=on', 'rpm.rpmva=on',
                               'rpm.rpmdb=on'])
```

Reply:
Alternatively, if this may be expanded again later, save the `plugopts` in another constant:

```python
CB_PLUG_OPTS = [
    'boot.all-images=on',
    'rpm.rpmva=on',
    'rpm.rpmdb=on'
]

CB_OPTS = SoSOptions(verify=True, all_logs=True, onlyplugins=_cb_plugs,
                     plugopts=CB_PLUG_OPTS)
```

Reply:
Oh sorry I didn't see your second post. I corrected the indentation like your first comment.